contents/product/correlation-analysis.mdx

```diff
@@ -1,25 +1,25 @@
 ---
-title: Diagnose causes with Correlation Analysis
-featuredImage: ../images/product/quantitative-analysis.png
+title: Diagnose causes automatically with Correlation Analysis
+featuredImage: ../images/apps/quantitative-analysis.png
 ---
 
 <Section
     divider={false}
-    title="Correlation Analysis automatically suggests which events or user properties lead to success or failure using transparent, testable, statistical models."
+    title="Correlation Analysis suggests which events or properties lead to success or failure using transparent statistical models."
```

Comment:
```suggestion
    title="Correlation Analysis suggests which events and properties lead to success or failure using transparent statistical models."
```
To avoid repeating 'or'